tests/spd/test_tlm_dlm_export_and_compile.py

```diff
@@ -0,0 +1,172 @@
+# -----------------------------------------------------------------------------
+#
+# Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# -----------------------------------------------------------------------------
+
+from typing import List, Optional
+
+import numpy as np
+import pytest
+from transformers import AutoTokenizer
+
+from QEfficient import QEFFAutoModelForCausalLM as AutoModelForCausalLM
+from QEfficient.generation.cloud_infer import QAICInferenceSession
+
+configs = [
+    pytest.param(
+        [0], # device_group
```

Comment:
is this triggered by the CI? would be interested in knowing how I can use this `AutoDevicePicker` so that I can run unit tests locally without having to change the code if I change `device_group` from `[0]` to `None`